src/core/lib/event_engine/tcp_socket_utils.cc

```diff
@@ -292,6 +337,17 @@ absl::optional<int> ResolvedAddressIsWildcard(
   }
 }
 
+#ifdef GRPC_HAVE_VSOCK
```

Comment:
I think this can be simpler

```
bool ResolvedAddressIsVSock(const EventEngine::ResolvedAddress& resolved_addr) {
#ifdef GRPC_HAVE_VSOCK
  return resolved_addr.address()->sa_family == AF_VSOCK;
#endif
  (void)resolved_addr;
  return false;
}
```